homeassistant/components/homekit/accessories.py

```diff
@@ -236,6 +245,18 @@ def __init__(self, hass, driver, name):
     def setup_message(self):
         """Prevent print of pyhap setup message to terminal."""
 
+    def get_snapshot(self, info):
```

Comment:
When is this called? A camera is a `HomeAccessory` now and has its `get_snapshot` implemented directly on it so can this code actually go away?

Reply:
The hap server calls `get_snapshot` on its "accessory handler" accessory. In our case that's the bridge. The bridge needs to call `get_snapshot` on the actual accessory (camera) if it exists. https://github.com/ikalchev/HAP-python/blob/51cb47322a33fd77df7e365801d6d6ddd5ac603a/pyhap/hap_server.py#L613

Reply:
Ah of course. I don't think it has to block this PR, but it feels like that function might be generic HAP enough to be better off upstream eventually too. Would you mind sending in another PR for that?